p2p/src/net/libp2p/proto/mdns.rs

```diff
@@ -209,7 +201,7 @@ mod tests {
                     Some(_) => {}
                     None => panic!("got None"),
                 },
-                event = backend2.swarm.next() => {}
+                _ = backend2.swarm.next() => {}
```

Comment:
It's preferable to just prefix variables with _ instead of just using _ as variable name. This helps in the future if someone needs to extend something and use the result. This is not particular to this line, but I think lots of _ where meaningful information could've been conveyed.

Reply:
I agree with @TheQuantumPhysicist in general. However, be aware that there is a subtle difference. Single `_` is a wildcard pattern [1] while `_foo` is an identifier pattern [2]. They differ in when is the value bound to it dropped. Adding a descriptive name can unnecessarily extend the lifetime of the value if the type implements `Drop`.

[1] https://doc.rust-lang.org/stable/reference/patterns.html#wildcard-pattern
[2] https://doc.rust-lang.org/stable/reference/patterns.html#identifier-patterns